packages/bottender-dialogflow/src/index.ts

```diff
@@ -30,7 +30,7 @@ function getFulfillments(fulfillmentMessages: Message[]): string[] {
  *   },
  * });
  */
-module.exports = function dialogflow({
+export default function dialogflow({
```

Comment:
This should be `export = function dialogflow({` in TS

Using `export default` here will introduce a breading change to commonjs users:

```js
const dialogflow = require('@bottender/dialogflow').default;
```

Reply:
Hi doing `export = function dialogflow` did not work.

Did you mean
`export function dialogflow({`

Reply:
What does `did not work` mean?

Actually, we have done this in release/1.5 branch.

https://github.com/Yoctol/bottender/blob/v1.5.1-alpha.5/packages/bottender-dialogflow/src/index.ts#L56

Reply:
Oh, Superb.
Did not know that.

Will close this PR.